code/workspaces/infrastructure-api/src/routers/clustersRouter.js

```diff
@@ -0,0 +1,20 @@
+import { service } from 'service-chassis';
+import express from 'express';
+import { permissionTypes } from 'common';
+import clustersController from '../controllers/clustersController';
+import permissionMiddleware from '../auth/permissionMiddleware';
+
+const { projectPermissions: { PROJECT_KEY_STACKS_CREATE } } = permissionTypes;
+
+const { errorWrapper } = service.middleware;
+
+const clustersRouter = express.Router();
+
+clustersRouter.post(
+  '/cluster',
```

Comment:
At the moment the route for creating a new cluster is `/clusters/cluster`. I think it would be good to just be a POST to `/clusters` by setting this to `'/'`. Unless there is a reason to have the additional `/cluster`?

Reply:
Agreed.